### DISCUSSION TITLE
Can screen be always on?

### DISCUSSION THREAD

**Discussion by mu88** [Quill-OS/quill#46] [Q&A] [answered]
Hi 👋🏻

I'm looking for a way to display a dashboard of my smart home solution openHAB on an E-reader and stumbled across your project.

For this, I need to display a website based on Vue.js. Does inkbox support the screen to be always on? Or will it always turn off after some time. 

Thanks!

**Comment by Szybet** [comment, marked as answer]
it can be set in the settings of the "InkBox power daemon"

You should more worry about showing the website, i don't know if midori browser will launch it, someone launched an old version of firefox on a kobo, maybe firefox will better handle javascript

**Reply by mu88** [reply]
Are there any specs about the browser so that I could check? Or do you have further material for reading?

**Reply by Szybet** [reply]
just install midori and try, if it will not work i will try to search for the guide for firefox

**Reply by mu88** [reply]
Do you know any way to test this beforehand, e. g. via an emulator of midori or inkbox? Because I'd need to buy a device for inkbox and if possible, I would like to know before whether it'll work 😉

**Reply by Szybet** [reply]
> Do you know any way to test this beforehand, e. g. via an emulator of midori or inkbox? Because I'd need to buy a device for inkbox and if possible, I would like to know before whether it'll work 😉

Just run midori and check ram usage of it ( ereaders don't have much ram ) @tux-linux is the latest version of midori used in inkbox?

**Reply by mu88** [reply]
On a Linux desktop, e. g. Raspberry Pi you mean?

**Reply by tux-linux** [reply]
> > Do you know any way to test this beforehand, e. g. via an emulator of midori or inkbox? Because I'd need to buy a device for inkbox and if possible, I would like to know before whether it'll work 😉
> 
> Just run midori and check ram usage of it ( ereaders don't have much ram ) @tux-linux is the latest version of midori used in inkbox?

It is the latest available as of 6/2022.

**Reply by Szybet** [reply]
> On a Linux desktop, e. g. Raspberry Pi you mean? 

Yep, any linux system

If ram doesnt exceed the 80% of the ereader ram, then you should be fine

Btw we don't promise it will work for your website on the ereader, no responsibility on us.

**Reply by dimatx** [reply]
@mu88 were you successful in having a website launch on boot in full screen mode? Looking to try to use it for a Home Assistant dashboard. I'd serve a PNG (to avoid it having to serve a more complex page with JS), and would want the device to refresh every so often (say every few minutes). Thoughts on whether this is achievable based on your tinkering?

**Reply by mu88** [reply]
@dimatx: I didn't continue this approach. Instead, I launched my own little project, see here: https://github.com/mu88/ScreenshotCreator/
